docs/mozillavpn_stage.json

```diff
@@ -14,6 +14,16 @@
     "WINDOWS": "2.7.1"
   },
   "releases": {
+    "2.0.0": {
+      "build": 1,
```

Comment:
These build numbers are fudged because I don't know what to put here and I'm not sure if it's relevant.